cmd/tools/migration/meta/210_to_220.go

```diff
@@ -176,6 +210,31 @@ func combineToSegmentIndexesMeta220(segmentIndexes SegmentIndexesMeta210, indexB
 	return segmentIndexModels, nil
 }
 
+func combineToLoadInfo220(collectionLoadInfo CollectionLoadInfo220, partitionLoadInto PartitionLoadInfo220, fieldIndexes FieldIndexes210) {
+	for collectionID, loadInfo := range collectionLoadInfo {
+		indexes, ok := fieldIndexes[collectionID]
+		if !ok {
+			delete(collectionLoadInfo, collectionID)
```

Comment:
I think this behavior needs more discussion. Should we warn here or fill an flat index meta? My concern is that if users loaded collections without index in the earlier version, in fact we break the compatiblity here and the users need to re-index the collection and then reload the collection.